inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-base/src/main/java/org/apache/inlong/sort/formats/inlongmsg/AbstractInLongMsgFormatDeserializer.java

```diff
@@ -81,6 +88,21 @@ public List<InLongMsgWrap> preParse(byte[] bytes) throws Exception {
         final List<InLongMsgWrap> result = new ArrayList<>();
 
         InLongMsg inLongMsg = InLongMsg.parseFrom(bytes);
+        if (inLongMsg == null) {
+            failureHandler.onParsingMsgFailure(bytes, new IOException(
+                    String.format("Could not parse InLongMsg from bytes. bytes={}.",
+                            StringUtils.join(bytes))));
+            return result;
+        }
+        try {
+            Set<String> set = inLongMsg.getAttrs();
+        } catch (Exception e) {
+            failureHandler.onParsingMsgFailure(bytes, new IOException(
+                    String.format("Parse InLongMsg from bytes has exception. bytes={}.",
+                            StringUtils.join(bytes)),
```

Comment:
ditto

Reply:
This is the callback processing, no printing